src/StreamJsonRpc/MessageFormatterHelper.cs

```diff
@@ -0,0 +1,31 @@
+﻿
+namespace StreamJsonRpc
+{
+    using System;
+    using System.Linq;
+    using System.Reflection;
+
+    /// <summary>
+    /// Class containing useful method to help on the implementation of message formatters.
+    /// </summary>
+    public class MessageFormatterHelper
+    {
+        /// <summary>
+        /// Converts given <see cref="Type"/> to its IProgress <see cref="Type"/>.
+        /// </summary>
+        /// <param name="objectType">The type implementing IProgress. </param>
+        /// <returns>The IProgress <see cref="Type"/> from given <see cref="Type"/> object.</returns>
```

Comment:
>. [](start = 90, length = 1)

, or <c>null</c> if no such interface was found in the given <paramref name="objectType" />.

Reply:
It looks like my first comment explaining how to enhance this comment was hiding the xml. I think I fixed it. Can you verify it reads better and re-make that change?

---
In reply to: [303034408](https://github.com/milopezc/vs-streamjsonrpc/pull/1#discussion_r303034408) [](ancestors = 303034408)